src/types.ts

```diff
@@ -15,8 +15,9 @@ export interface Video {
 }
 
 export interface MessageTransport {
-    log: (message: any) => void,
+    log: (message: any) => void
     userAction: (message: string) => void
+    prompt?: (message: string) => Promise<string>
```

Comment:
Thanks ! I actually wanted to make a similar merge request.
I think we could rename this callback to something more specific.
Maybe to "onSmsVerificationCodeSent" ? "promptSmsVerificationCode", "challengeSmsVerificationCodeHandler" or something along these lines ?

The reason I think it has to be specific, is because for my own instagram related automations, I used a callback specific to verification codes to then feed that into a telegram channel where I'm able to input the code.
https://github.com/pierreminiggio/instagram-story-poster/blob/454dc3bc20d430b702db1efa7273372a305b4937/upload_story.py#L16-L31

Reply:
Makes sense, will update it!